lib/ansible/plugins/vars/sops_vars.py

```diff
@@ -0,0 +1,258 @@
+# Copyright 2019 Arduino, srl
```

Comment:
Please use instead:
```suggestion
# -*- coding: utf-8 -*-

# Copyright: (c) 2019, Arduino, srl
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```
Plugins need to be GPLv3+ licensed. Above is the (short) form usually used.